get-resource.sh

```diff
@@ -41,9 +41,14 @@ else
 
     for i in $(seq ${MAX_ATTEMPTS}); do
         if ! curl -g --insecure --compressed -L --connect-timeout ${CONNECT_TIMEOUT} -o "${RHCOS_IMAGE_FILENAME_RAW}" "${IMAGE_URL}/${RHCOS_IMAGE_FILENAME_RAW}"; then
-          SLEEP_TIME=$((i*i))
-          echo "Download failed, retrying after ${SLEEP_TIME} seconds..."; 
-          sleep ${SLEEP_TIME}
+          if (( ${i} == ${MAX_ATTEMPTS} )); then
```

Comment:
Is it really a correct format? I think you need square brackets:
```bash
$ if $(( 1 == 2 )); then
>  echo OK
> fi
bash: 0: command not found
```

Reply:
@dtantsur  no $ after if
$ if (( 1 == 2 )); then 
> echo OK
> fi
$ if (( 1 == 1 )); then  echo OK; fi
OK
`

Reply:
Ah, okay. I haven't seen this form yet.